instrumentation/instrumentation_test.go

```diff
@@ -16,6 +16,12 @@ func TestInstrumentation_AnalyticsRate(t *testing.T) {
 	pkgs := GetPackages()
 	for pkg, info := range pkgs {
 		t.Run(string(pkg), func(t *testing.T) {
+			// Skip packages that don't implement analytics functionality
```

Comment:
Let me know if this kind of skip is acceptable. Currently the datadog-lambda-go doesn't support instrumentation telemetry